README

```diff
@@ -292,9 +292,12 @@ WARNINGS:
         https://www.freedesktop.org/wiki/Software/systemd/separate-usr-is-broken
 
         To run systemd under valgrind, compile with VALGRIND defined
-        (e.g. ./configure CPPFLAGS='... -DVALGRIND=1'). Otherwise,
-        false positives will be triggered by code which violates
-        some rules but is actually safe.
+        (e.g. ./configure CPPFLAGS='... -DVALGRIND=1') and have valgrind
```

Comment:
`CPPFLAGS=... meson build`

http://mesonbuild.com/howtox.html#set-extra-compiler-and-linker-flags-from-the-outside-when-eg-building-distro-packages